Data/Map/Internal.hs

```diff
@@ -1981,8 +1981,8 @@ intersection t1@(Bin _ k x l1 r1) t2
 -- found in a 'Set'.
 --
 -- @
--- m `'restrictKeys'` s = 'filterWithKey' (\k _ -> k `'Set.member'` s) m
--- m `'restrictKeys'` s = m `'intersect' 'fromSet' (const ()) s
+-- m \`restrictKeys\` s = 'filterWithKey' (\k _ -> k ``Set.member`` s) m
+-- m \`restrictKeys\` s = m ``intersect`` 'fromSet' (const ()) s
```

Comment:
What's going on here? Backslash on the left, double ticks on the right?

Reply:
This is in the documentation for restrictKeys. Just as I said: “I went with making everything links […], except for in a functions documentation for the identifier of that function itself (that link would take you nowhere else)”.